modules/ocf_irc/manifests/ircd.pp

```diff
@@ -8,10 +8,10 @@
   }
 
   ocf::systemd::override { 'start-inspircd-before-anope':
-    service => 'inspircd',
-    content => "[Unit]\nBefore=anope.service",
+    unit    => 'inspircd.service',
+    content => "[Unit]\nBefore=anope.service\n",
     require => Package['inspircd', 'anope'],
-    before  => Service['inspircd'],
+    notify  => Service['inspircd'],
```

Comment:
Well, `notify` implies `before`, but I did touch upon your point in your previous comment.